src/aviary/env.py

```diff
@@ -39,6 +39,13 @@
 Serializable: TypeAlias = dict | list | int | float | str | bool | BaseModel
 
 
+async def maybe_wait_for(future: Awaitable, timeout: float | None) -> Any:
```

Comment:
I made https://github.com/Future-House/aviary/pull/143 to type this, I missed the boat on this PR review